### PR TITLE
iOS install instructions for Edge Network extension users in Launch

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: current-sdk-versions.md [hunk 1]
```diff
@@ -2,38 +2,32 @@
 
 The Adobe Experience Platform Mobile SDK supports the following OS versions:
 
-* Google Android API 14 \(Ice Cream Sandwich\) or later
+* Google Android API 16 or later
```
Comment (kevinlind) [inline current-sdk-versions.md:5]: Should we add a note that Edge Android requires minimum API 19?

File: installation-guide.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+# iOS AEP SDKs installation guide
+
+## Migration considerations 
+
+Here are a few things to consider when installing the Adobe Experience Platform Edge Network mobile extension.
+
+- The new AEP Edge Network extension requires the usage of the latest AEP SDKs, which for iOS is a new Swift-based SDK. If you are an ACP Mobile SDK user, read more about this in the [Adobe Experience Platform Edge Network Migration Considerations](../getting-started/introduction-to-aepedge.md#adobe-experience-platform-edge-network-migration-considerations).
+- For the latest information on supported Swift SDK compatible extensions, see [Current SDK Versions](current-sdk-versions.md).
+- This extension requires the use of [Identity for Edge Network mobile extension](../using-mobile-extensions/adobe-edge-identity).
```
Comment (kevinlind) [inline installation-guide.md:9]: Let's remove `this`, "The AEP Edge Network mobile extension requires the ..."

File: installation-guide.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+- When using this extension, it is strongly encouraged to use Consent mobile extension for managing and enforcing the collect consent settings. For more details see [Consent extension](../using-mobile-extensions/adobe-edge-consent) and [Privacy and GDPR](../privacy-and-gdpr.md).
```
Comment (kevinlind) [inline installation-guide.md:10]: Let's remove `this`. "When using the AEP Edge Network mobile extension, it is strongly ..."

File: installation-guide.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
+# iOS AEP SDKs installation guide
+
+## Migration considerations 
+
+Here are a few things to consider when installing the Adobe Experience Platform Edge Network mobile extension.
+
+- The new AEP Edge Network extension requires the usage of the latest AEP SDKs, which for iOS is a new Swift-based SDK. If you are an ACP Mobile SDK user, read more about this in the [Adobe Experience Platform Edge Network Migration Considerations](../getting-started/introduction-to-aepedge.md#adobe-experience-platform-edge-network-migration-considerations).
```
Comment (kevinlind) [inline installation-guide.md:7]: I think you can remove the "more about this in the" and just say "If you are an ACP Mobile SDK user, read Adobe Experience Platform Edge Network Migration Considerations."

**Review by shalehaha — APPROVED** [review]

**Review by kevinlind — APPROVED** [review]

**Pull request merged**